cluster-autoscaler/core/utils.go

```diff
@@ -67,9 +67,10 @@ const (
 // UID as a key in initial lookup and only run full comparison on a set of
 // podSchedulableInfos created for pods owned by this controller.
 type podSchedulableInfo struct {
-	spec        apiv1.PodSpec
-	labels      map[string]string
-	schedulable bool
+	spec            apiv1.PodSpec
+	labels          map[string]string
+	schedulable     bool
```

Comment:
Do we need a bool field if we have error? I assume `schedulable == (schedulingError == nil)` is always going to be true?

Reply:
Ok, I see we don't always use it. Still I sort of don't like the resulting semantics What would be the meaning of scheduled==true and schedulingError!=nil? Maybe just have a singleton throwaway PredicateError and use this if we don't care about actual error?

Reply:
I thought having two fields is the lesser evil (i.e. we won't have to use artificial predicate error), but I can change this if you think it's better.

Reply:
Removed the field as suggested, PTAL